headers/types/ground_mode/live_actor.h

```diff
@@ -0,0 +1,94 @@
+#ifndef HEADERS_LIVE_ACTOR_GROUND_MODE_H_
+#define HEADERS_LIVE_ACTOR_GROUND_MODE_H_
+
+struct LiveActor {
+    uint16_t id;     /* The id of the Actor in the actor list */
+    uint16_t kind;   /* The specie of the Actor */
```

Comment:
specie -> species. Also, is this the monster ID, or something else? If it's the monster ID, let's use `struct monster_id_16` as the type.

Reply:
> Also, is this the monster ID, or something else?

So is the answer to this question no? `struct monster_id_16` isn't appropriate here?